takwimu/takwimu_ui/src/components/DataContainer/EntitiesDataContainer.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable react/no-danger */
+import React, { useState, useEffect, Fragment } from 'react';
+import { PropTypes } from 'prop-types';
+
+import { withStyles } from '@material-ui/core';
+
+import domtoimage from 'dom-to-image';
+import DataActions from './DataActions';
+
+const styles = {
+  root: {
+    width: '100%'
+  }
+};
+
+function DataContainer({ id, classes, data }) {
+  const [images, setImages] = useState({});
+
+  useEffect(() => {
+    data.entities.forEach(
+      entity =>
+        entity.image &&
+        fetch(`/api/v2/images/${entity.image}`)
```

Comment:
What is happening with this fetch? Aren't URLs returned as part of the API i.e. https://docs.wagtail.io/en/v2.3/advanced_topics/api/v2/configuration.html#images-in-the-api? If not, please create a PT story and PR for that.

Reply:
The fetch is returning Ids.
I had tried the API Field but It did not seem to work.
I'll create a PT story might need to explore slightly more than I did.